src/main/java/com/redhat/sast/api/service/MlOpsBatchService.java

```diff
@@ -0,0 +1,204 @@
+ * Service for managing MLOps batch jobs.
+ * Handles submission and processing of MLOps pipeline batches.
+ */
+@ApplicationScoped
+@Slf4j
+public class MlOpsBatchService extends AbstractBatchService {
+
+    private final ManagedExecutor managedExecutor;
+    private final PackageListResolver packageListResolver;
+
+    // No-args constructor required by Quarkus CDI for proxying
+    protected MlOpsBatchService() {
+        super(null, null, null, null, null);
+        this.managedExecutor = null;
+        this.packageListResolver = null;
+    }
+
+    public MlOpsBatchService(
+            JobBatchRepository jobBatchRepository,
+            JobService jobService,
+            PlatformService platformService,
+            PipelineParameterMapper parameterMapper,
+            BatchProcessingService batchProcessingService,
+            ManagedExecutor managedExecutor,
+            PackageListResolver packageListResolver) {
+        super(jobBatchRepository, jobService, platformService, parameterMapper, batchProcessingService);
+        this.managedExecutor = managedExecutor;
+        this.packageListResolver = packageListResolver;
+    }
+
+    /**
+     * Submits an MLOps batch job for processing.
+     */
+    public JobBatchResponseDto submitMlOpsBatch(MlOpsJobBatchSubmissionDto submissionDto) {
+        JobBatch batch = createInitialMlOpsBatch(submissionDto);
+
+        JobBatchResponseDto response = convertToResponseDto(batch);
+
+        managedExecutor.execute(() -> executeMlOpsBatchProcessing(
+                batch.getId(),
+                batch.getUseKnownFalsePositiveFile(),
+                batch.getSubmittedBy(),
+                submissionDto.getDvcNvrVersion(),
+                submissionDto.getDvcKnownFalsePositivesVersion(),
+                submissionDto.getDvcPromptsVersion(),
+                submissionDto.getImageVersion()));
+
+        return response;
+    }
+
+    @Transactional
+    public JobBatch createInitialMlOpsBatch(MlOpsJobBatchSubmissionDto submissionDto) {
+        JobBatch batch = new JobBatch();
+        batch.setSubmittedBy(submissionDto.getSubmittedBy() != null ? submissionDto.getSubmittedBy() : "unknown");
+        batch.setUseKnownFalsePositiveFile(submissionDto.getUseKnownFalsePositiveFile());
+        batch.setStatus(BatchStatus.PROCESSING);
+        batch.setPipelineName(PipelineConstants.MLOPS_PIPELINE);
+
+        jobBatchRepository.persist(batch);
+        LOGGER.info("Created new MLOps job batch with ID: {}", batch.getId());
+
+        return batch;
+    }
+
+    /**
+     * Asynchronously processes an MLOps batch by fetching package list and creating individual jobs.
+     */
+    public void executeMlOpsBatchProcessing(
+            @Nonnull Long batchId,
+            Boolean useKnownFalsePositiveFile,
+            String submittedBy,
+            String dvcNvrVersion,
+            String dvcKnownFalsePositivesVersion,
+            String dvcPromptsVersion,
+            String imageVersion) {
```

Comment:
Let's create a record called "MlOpsConfig" to store these values. Because I can see we can reuse that record again as method inputs in `processMlOpsJobs(MlOpsConfig config, List<MlOpsJobCreationDto> jobDtos)` in this same class.